Integrations/GitHub/GitHub.py

```diff
@@ -128,148 +119,127 @@ def List_all_issues(show_all):
 def List_all_issues_command():
     show_all = demisto.args()['show-all']
     issues = List_all_issues(show_all)
-    issue_list = []
+    issue_list = []  # type : dict
     for s in issues:
         context = get_issue_context(s)
         issue_list.insert(0, context)
 
     header_list = ['ID', 'title', 'state', 'locked', 'body', 'assignees', 'labels']
-    human_readable = tableToMarkdown('issues list: ', issue_list, header_list)
+    human_readable = tableToMarkdown('issue list', issue_list, header_list)
     entry_context = {
         'Git.issue(val.ID && val.ID == obj.ID)': issue_list
     }
 
     return_outputs(human_readable, entry_context, issues)
 
 
-def create_issue(args):
-    suffix = '/repos/{}/{}/issues'.format(OWNER, REPO)
-    data = {}
+def create_or_update_issue(command):
+    args = demisto.args()
     for key in args.keys():
-        data[key] = args[key]
-
-    if 'labels' in data:
-        data['labels'] = data['labels'].split(',')
-
-    if 'assignees' in data:
-        data['assignees'] = data['assignees'].split(',')
+        if key == 'labels':
+            args['labels'] = args['labels'].split(',')
+        if key == 'assignees':
+            args['assignees'] = args['assignees'].split(',')
+    args = json.dumps(args)
+    if command == 'create':
+        suffix = SUFFIX['create']
+        res = http_request('POST', suffix, json=args)
+    else:
+        suffix = SUFFIX['update'] + demisto.args()['issue_id']
+        res = http_request('PATCH', suffix, args)
 
-    data = json.dumps(data)
-    # return_error(data)
-    res = http_request('POST', suffix, data, None)
     return res
 
 
 def create_issue_command():
-    args = demisto.args()
-    res = create_issue(args)
+    res = create_or_update_issue('create')
     context = get_issue_context(res)
 
     header_list = ['ID', 'title', 'state', 'locked', 'body', 'assignees', 'labels']
-    human_readable = tableToMarkdown('created issue successfully: ', context, header_list, removeNull=True)
+    human_readable = tableToMarkdown('created issue successfully', context, header_list, removeNull=True)
     entry_context = {
-        'Git.issue(val.id && val.id == obj.id)': context
+        'Git.issue(val.ID && val.ID == obj.ID)': context
     }
-
     return_outputs(human_readable, entry_context, res)
 
 
-def close_issue(issue_id):
-    suffix = '/repos/{}/{}/issues/{}'.format(OWNER, REPO, issue_id)
+def close_issue():
+    suffix = SUFFIX['close'] + demisto.args()['issue_id']
     data = {"state": "close"}
-
     data = json.dumps(data)
-    res = http_request('PATCH', suffix, data, None, 'close')
+    res = http_request('PATCH', suffix, data, params='close')
```

Comment:
shouldn't the params be a dictionary?

Reply:
This shouldn't be here, something I tried to do in the previous version. 
This specific request only uses the data argument so it didn't make any difference in the command behaviour. Removed